WalletWasabi/Blockchain/Transactions/Summary/InputAmount.cs

```diff
@@ -0,0 +1,13 @@
+using NBitcoin;
+
+namespace WalletWasabi.Blockchain.Transactions.Summary;
+
+public class InputAmount : Input
+{
+	public InputAmount(Money amount, BitcoinAddress address) : base(amount)
```

Comment:
Is the naming correct here? For me calling this class `Input` and calling the base class `InputAmount` makes more sense as the base is only about the amount.

Reply:
Please, check the class names again. I've tried to make them better. I wouldn't like to call them *Amount because it's not only the amount, but more data. It corresponds to the concept of "source transaction". I'm omitting the "transaction" part because it's already implicit.